backend/src/main/java/saci/application/mappers/AnswerMapper.java

```diff
@@ -0,0 +1,20 @@
+package saci.application.mappers;
+
+import saci.application.model.ScoreRequest.Answer;
+import saci.domain.model.Knowledge;
+
+public class AnswerMapper {
+    public static Knowledge toDomain(Answer answer) {
```

Comment:
If this class is the AnswerMapper this method should be in a KnowledgeMapper class because it returns a Knowledge.